.github/workflows/mobile-ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Mobile CI
+
+on:
+  push:
```

Comment:
Please add specs  **paths:** into push to ensure code change related to merchant app mobile the github actions flow will be trigger CI.